interface/.dockerignore

```diff
@@ -0,0 +1,4 @@
+bin
+build
+node_modules
```

Comment:
Yes, yes you are :)

That sounds like a really good plan. It's a similar strategy to the one I'm using in the Data Refinery for its dependencies. They take a long time to run so I've hosted a base image [here](https://hub.docker.com/r/ccdl/data_refinery_worker_base/) so that if someone wants to develop on the Data Refinery they don't have to first wait for 2 hours while the image is built on their machine and so that CI can run in a reasonable amount of time.